### PR TITLE
created dark, light and gray theme

### PR DESCRIPTION
**Pull request by akanksha-patel-code**
# Description

Created themes for Dark, light and gray mode.

## Type of change

- ✅ new feature

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@
 
 ## [Unreleased]
 
+## [0.0.3]
```
Comment (bhushan) [inline CHANGELOG.md]: remove this tag, whenever we release new version, whatever inside unreleased will be tagged automatically

File: src/main/kotlin/com/github/easternenterprise/jetbrainstheme/listeners/MyProjectManagerListener.kt [hunk 2]
```diff
@@ -1,6 +1,6 @@
-package com.github.easternenterprise.githubtheme.listeners
+package com.github.easternenterprise.jetbrainstheme.listeners
```
Comment (bhushan) [inline src/main/kotlin/com/github/easternenterprise/jetbrainstheme/listeners/MyProjectManagerListener.kt]: This file is not required IMO

File: src/main/kotlin/com/github/easternenterprise/jetbrainstheme/services/MyApplicationService.kt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.github.easternenterprise.jetbrainstheme.services
```
Comment (bhushan) [inline src/main/kotlin/com/github/easternenterprise/jetbrainstheme/services/MyApplicationService.kt]: This file is also not required IMO

File: src/main/kotlin/com/github/easternenterprise/jetbrainstheme/services/MyProjectService.kt [hunk 4]
```diff
@@ -1,6 +1,6 @@
-package com.github.easternenterprise.githubtheme.services
+package com.github.easternenterprise.jetbrainstheme.services
```
Comment (bhushan) [inline src/main/kotlin/com/github/easternenterprise/jetbrainstheme/services/MyProjectService.kt]: this one as well

File: src/main/kotlin/com/github/eebhushan/githubtheme/services/MyApplicationService.kt [hunk 5]
```diff
@@ -1,10 +0,0 @@
-package com.github.easternenterprise.githubtheme.services
```
Comment (bhushan) [inline src/main/kotlin/com/github/eebhushan/githubtheme/services/MyApplicationService.kt:1]: this one as well

File: src/main/resources/Jetbrains_Dark_Dimmed.theme.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "JetBrains Dark Dimmed",
```
Comment (bhushan) [inline src/main/resources/Jetbrains_Dark_Dimmed.theme.json]: There is gray already, It should be dark only. `gray === dark dimmed`

File: src/main/resources/Jetbrains_Dark_Dimmed.theme.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  "dark": true,
+  "author": "Akanksha <a.patel@easternenterprise.com>",
+  "editorScheme": "/Jetbrains_Dark_Dimmed.xml",
```
Comment (bhushan) [inline src/main/resources/Jetbrains_Dark_Dimmed.theme.json]: Change files names and paths accordingly

File: src/main/resources/Jetbrains_Dark_Dimmed.xml [hunk 8]
```diff
@@ -0,0 +1,575 @@
+<scheme name="GitHub Dark Dimmed" version="142" parent_scheme="Darcula">
```
Comment (bhushan) [inline src/main/resources/Jetbrains_Dark_Dimmed.xml]: Change schema names accordingly

File: src/main/resources/Jetbrains_Dark_Dimmed.xml [hunk 9]
```diff
@@ -0,0 +1,575 @@
...
+    <metaInfo>
+        <property name="created">2021-08-21T08:57:01</property>
+        <property name="ide">idea</property>
+        <property name="ideVersion">2021.2.0.0</property>
+        <property name="modified">2021-08-21T08:57:18</property>
+        <property name="originalScheme">_@user_GitHub Dark Dimmed</property>
```
Comment (bhushan) [inline src/main/resources/Jetbrains_Dark_Dimmed.xml]: change schema name accordingly

File: src/main/resources/Jetbrains_Gray.theme.json [hunk 10]
```diff
@@ -0,0 +1,121 @@
+{
+  "name": "JetBrains Gray",
+  "dark": false,
```
Comment (bhushan) [inline src/main/resources/Jetbrains_Gray.theme.json]: gray mode  is part of dark mode, Its our way of saying lighter dark mode, so it should be true

File: src/main/resources/Jetbrains_Light.xml [hunk 11]
```diff
@@ -0,0 +1,1090 @@
+<scheme name="GitHub Light" version="142" parent_scheme="Default">
```
Comment (bhushan) [inline src/main/resources/Jetbrains_Light.xml]: change schema name

File: src/main/resources/Jetbrains_Light.xml [hunk 12]
```diff
@@ -0,0 +1,1090 @@
...
+    <metaInfo>
+        <property name="created">2021-08-21T14:38:17</property>
+        <property name="ide">Idea</property>
+        <property name="ideVersion">2021.2.0.0</property>
+        <property name="modified">2021-08-21T14:38:23</property>
+        <property name="originalScheme">_@user_GitHub Light</property>
```
Comment (bhushan) [inline src/main/resources/Jetbrains_Light.xml]: same

File: src/main/resources/META-INF/plugin.xml [hunk 13]
```diff
@@ -1,19 +1,40 @@
 <idea-plugin>
-    <id>com.github.easternenterprise.githubtheme</id>
+    <id>com.github.easternenterprise.jetbrainstheme</id>
     <name>GitHub-Theme</name>
```
Comment (bhushan) [inline src/main/resources/META-INF/plugin.xml]: Change Theme name

File: src/main/resources/META-INF/plugin.xml [hunk 14]
```diff
@@ -1,19 +1,40 @@
...
     <vendor url="https://github.com/easternenterprise/github-theme">Eastern Enterprise</vendor>
```
Comment (bhushan) [inline src/main/resources/META-INF/plugin.xml]: change url to `https://github.com/easternenterprise` this

File: src/main/resources/META-INF/plugin.xml [hunk 15]
```diff
@@ -1,19 +1,40 @@
...
 
+    <description><![CDATA[
+  GitHub Theme
```
Comment (bhushan) [inline src/main/resources/META-INF/plugin.xml]: change name here

File: src/main/resources/META-INF/plugin.xml [hunk 16]
```diff
@@ -1,19 +1,40 @@
...
 
+
     <extensions defaultExtensionNs="com.intellij">
-        <applicationService serviceImplementation="com.github.easternenterprise.githubtheme.services.MyApplicationService"/>
-        <projectService serviceImplementation="com.github.easternenterprise.githubtheme.services.MyProjectService"/>
+        <applicationService serviceImplementation="com.github.easternenterprise.jetbrainstheme.services.MyApplicationService"/>
+        <projectService serviceImplementation="com.github.easternenterprise.jetbrainstheme.services.MyProjectService"/>
+        <!-- Add your extensions here -->
+        <themeProvider id="adf99807-aee9-4514-b512-5f0e429f30da" path="/Jetbrains_Light.theme.json"/>
+        <themeProvider id="776bbe10-21be-477e-a5b3-5cb4403feab4" path="/Jetbrains_Dark_Dimmed.theme.json"/>
```
Comment (bhushan) [inline src/main/resources/META-INF/plugin.xml]: Please Correct file paths once you change names of the files

File: src/main/resources/META-INF/plugin.xml [hunk 17]
```diff
@@ -1,19 +1,40 @@
...
+        <themeProvider id="eb9b7461-397b-4b98-a422-224fc0a74564" path="/Jetbrains_Gray.theme.json"/>
     </extensions>
 
     <applicationListeners>
```
Comment (bhushan) [inline src/main/resources/META-INF/plugin.xml]: this listener section is not required IMO

**Review by bhushan — DISMISSED** [review]

File: src/main/kotlin/com/github/easternenterprise/jetbrainstheme/MyBundle.kt [hunk 18]
```diff
@@ -1,4 +1,4 @@
-package com.github.easternenterprise.githubtheme
+package com.github.easternenterprise.jetbrainstheme
```
Comment (ee-bhushan) [inline src/main/kotlin/com/github/easternenterprise/jetbrainstheme/MyBundle.kt]: `package com.easternenterprise.jetbrainstheme`
Change it like this

File: src/main/resources/Jetbrains_Dark.theme.json [hunk 19]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (ee-bhushan) [inline src/main/resources/Jetbrains_Dark.theme.json]: have you tested these themes ? are they looking good ? could you please share screenshots ?

Comment (ee-bhushan) [reply src/main/resources/Jetbrains_Dark.theme.json]: screenshots for all three variants

File: src/main/resources/META-INF/plugin.xml [hunk 20]
```diff
@@ -1,19 +1,34 @@
 <idea-plugin>
-    <id>com.github.easternenterprise.githubtheme</id>
-    <name>GitHub-Theme</name>
-    <vendor url="https://github.com/easternenterprise/github-theme">Eastern Enterprise</vendor>
+    <id>com.github.easternenterprise.jetbrainstheme</id>
```
Comment (ee-bhushan) [inline src/main/resources/META-INF/plugin.xml]: `<id>com.easternenterprise.jetbrainstheme</id>`

change it like this

File: src/main/resources/META-INF/plugin.xml [hunk 21]
```diff
@@ -1,19 +1,34 @@
...
+    <name>Jetbrain-Theme</name>
```
Comment (ee-bhushan) [inline src/main/resources/META-INF/plugin.xml:3]: Everywhere its `Jetbrains` keep it consistent

**Review by ee-bhushan — CHANGES_REQUESTED** [review]

File: src/main/resources/Jetbrains_Dark.theme.json (hunk 19 above)
Comment (akanksha-patel-code) [reply src/main/resources/Jetbrains_Dark.theme.json]: I have tested these three themes :
1 - Dark mood 
It is looking as expected.

2- Light mood
It is looking as expected

3- Dark mood
It is looking as expected

Please find the attached screenshot for your reference

Dark
Gray
Light

**Review by ee-bhushan — APPROVED** [review]

**Pull request merged**